apps/yarn.lock

```diff
@@ -2681,8 +2681,9 @@ acorn@^3.0.4:
   integrity sha1-ReN/s56No/JbruP/U2niu18iAXo=
 
 acorn@^5.5.0:
-  version "5.7.3"
-  resolved "https://registry.yarnpkg.com/acorn/-/acorn-5.7.3.tgz#67aa231bf8812974b85235a96771eb6bd07ea279"
+  version "5.7.4"
```

Comment:
Do you know why all of these changes in yarn.lock are needed?

Reply:
I'm not sure why those changes are being generated when I run `yarn`

Reply:
Update: I reverted yarn.lock and ran `yarn` again, now the `yarn.lock` file looks as expected. Not sure what I did wrong there before.